modules/developer_manual/examples/scripts/backport.sh

```diff
@@ -38,6 +36,9 @@ pullId=$(git log $1^! --oneline 2>/dev/null | tail -n 3 | grep -oP '(?<=#)[0-9]*
 # get the repository from commit
 repository=$(git config --get remote.origin.url 2>/dev/null | perl -lne 'print $1 if /(?:(?:https?:\/\/github.com\/)|:)(.*?).git/')
 
+# get the list of commits in PR without any merge commit
+commitList=$(git log --no-merges --reverse --format=format:%h $1^..$1)
```

Comment:
Note to other readers who (like me) might have to think about what this does: `$1^` means the first parent of the merge commit (that is passed in as `$1`). Because `$1` is a "magically-generated" merge commit, it happily "jumps back" to the point on the main branch just before where the PR was merged. And so the commits from that point are exactly the list of individual commits in the original PR.
`--no-merges` leaves out the merge commit itself, and we get just what we want - works nicely.

Reply:
Thanks for your feedback.
I have fixed the grammar from above and added your note as comment for other readers